pyanaconda/modules/boss/boss_interface.py

```diff
@@ -84,3 +84,7 @@ def DistributeKickstart(self) -> List[Dict[Str, Variant]]:
             "line_number": get_variant(Int, result["line_number"]),
             "error_message": get_variant(Str, result["error_message"])
         } for result in results]
+
+    def SetLocale(self, locale: Str):
+        """Set locale of all modules."""
```

Comment:
Please, document an example of the locale.

Reply:
Added.